src/plugins/typingTweaks.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Vencord, a modification for Discord's desktop app
+ * Copyright (c) 2023 Vendicated and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+import { findByCodeLazy } from "@webpack";
+import { GuildMemberStore, React } from "@webpack/common";
+
+const Avatar = findByCodeLazy(".Positions.TOP,spacing:");
+
+export default definePlugin({
+    name: "TypingTweaks",
+    description: "Adds some tweaks to typing indicators",
+    authors: [Devs.zt],
+    patches: [
+        {
+            find: "getCooldownTextStyle",
+            replacement: {
+                match: /=(.{1,2})\[2];(.+)"aria-atomic":!0,children:(.{1,2})}\)/,
```

Comment:
```suggestion
                match: /=(\i)\[2];(.+)"aria-atomic":!0,children:(\i)}\)/,
```
you can use \i to match identifiers instead of .{1,2}, recently added patcher feature